changelog_unreleased/html/11827.md

```diff
@@ -1 +1,26 @@
 #### Reverts #7865, keeping class names on the same line (#11827 by @jlongster)
```

Comment:
I think this should be a highlight since it is likely to substantially affect some users’ code.

```suggestion
#### Collapse HTML `class` attributes onto one line (#11827 by @jlongster, reverts #7865)
```

Also, a change like this should probably have a rationale. Here’s my proposal, but as I’m not an active maintainer, I think y’all’s opinions are more valuable.

> While this was intended to be useful for users of CSS libraries like Tailwind that tend to result in large numbers of classes on elements, it became clear that our heuristics for where to split the class list on to multiple lines were unable to consistently produce good results. We’re still considering better ways to format HTML with lots of classes — consider `[discussing with us](https://github.com/prettier/prettier/issues/7863)`.